### ISSUE TITLE
Release core-1.4.2.2 to support mtl-2.3

### ISSUE THREAD

**Issue by andreasabel** [UnkindPartition/tasty#333]
Release core-1.4.2.2 to support mtl-2.3.
I get to work.

**Comment by andreasabel** [comment]
I tested `mtl-2.3` compatibility by `cabal build core` with the following `cabal.project.local`:
```
constraints: mtl == 2.3
allow-newer: mtl

-- random
------------------------------------------------------------------------

source-repository-package
  type: git
  location: https://github.com/haskell/random.git
  tag: 5ecc8935994159c6d39088233a0887714aacaf5e

constraints: random == 1.3.*
allow-newer: *:random
``` 

The testsuites are still blocked by at least:
- [ ] https://github.com/byorgey/MonadRandom/issues/50
- [ ] https://github.com/Bodigrim/logict/issues/31

**Comment by andreasabel** [comment]
I realized I have no upload rights on hackage.  So I have to unassign myself.
The release preps are in this PR:
- #334

**Comment by kozross** [comment]
@andreasabel Any chance we could cut a Hackage release? The lack of such blocks me from giving you [the tests you requested](https://github.com/haskell/mtl/issues/117#issuecomment-1120492816).

**Comment by andreasabel** [comment]
@UnkindPartition : Would you mind adding me to the hackage maintainers of `tasty`?

**Comment by kozross** [comment]
@ocharles - As per @UnkindPartition's code will, would you be able to help out @andreasabel and myself?

**Comment by andreasabel** [comment]
I emailed Oliver Charles about it.

**Comment by andreasabel** [comment]
@kozross: I you want to get started with using `tasty` as testsuite for `mtl`, you can use the `no-mtl` branch (#335) per `source-repository` entry in your `cabal.project`.

**Comment by ocharles** [comment]
@andreasabel @kozross @UnkindPartition all done!

**Comment by andreasabel** [comment]
Released `tasty-1.4.2.2` to hackage.  
TODO:
- [ ] signed tag `core-1.4.2.2`

**Comment by andreasabel** [comment]
Closed by 
- #334.
